platform/linux-generic/arch/aarch64/cpu_flags.c

```diff
@@ -395,7 +1122,118 @@ static void _odp_sys_info_print_acle_flags(void)
 	ODP_PRINT("\n");
 }
 
+static int check_hwcap_duplicates(unsigned int hwcap_field)
+{
+	int ret = 0;
+
+	/* FP and AdvSIMD fields of the AArch64 Processor
+	 * Feature Register 0 must have the same value and are
+	 * defined by the same feature flag. Print the flag
+	 * only once. */
+#ifdef HWCAP_ASIMD
+		if (hwcap_field == HWCAP_ASIMD)
+			ret = 1;
+#endif
+#ifdef HWCAP_ASIMDHP
+		if (hwcap_field == HWCAP_ASIMDHP)
+			ret = 1;
+#endif
+
+	return ret;
+}
+
+static void _odp_sys_info_print_hwcap_flags(void)
+{
+	unsigned long hwcaps, hwcaps2;
+	unsigned int size, size2;
+
+	/* Check if the hwcap_flags[] is an empty array. If true, the function
+	 * will return. This is unlikely behaviour. */
+	size = sizeof(hwcap_flags) / sizeof(hwcap_feat_flag_t);
+	if (size == 0)
+		return;
```

Comment:
I guess  this return and the comment above can now be removed, since we also have the `ARM FEATURES NOT SUPPORTED BY KERNEL:` print and the array is never empty.

Reply:
I missed removing that in the code. Thanks Matias!